docs/computer_setup.md

```diff
@@ -2,82 +2,161 @@
+- Initialize the database (this creates an empty database)
+	- run `python ./scripts/create_database.py`
+	- In normal use, once you create the database, you will not need to do this again.
+	When testing, however, you may need to remove it and run it again.
+- To confirm the script ran successfully:
+	- rum `ls`
+	- If the file "sss.sqlite" is present, the create_database.py script did its job
+- To delete the file
+	- run `rm sss.sqlite`
 
 ## Running the script:
-* Call to insert data from a file or folder of interest, into an already existing primary_table database (this cannot be done unless the database has already been initialized):
-	* Type `python ./scripts/data_to_primary.py [path name]`
-* To find the path name (add)
-	* *NYC2018_SSS_Full.xlsx and NYC2021_SSS_Full.xlsx are omitted from the data folder that creates the primary table because they have repeating information that is already in the database*
+
+- First, we have to find the correct path name. This is not totally obvious on Windows
+using WSL because you can't just use the path to the SSS data files that you find from
+your Explorer file browser. Instead we need to find where the SSS files are on your
+machine according to the WSL terminal.
+	- The way we examine what files are inside a folder in the terminal is using the
+	`ls` command. If you run `ls` by itself it will print out all the files that are in
+	your current working directory. If you add a path after `ls` it will print the
+	files that are inside that path.
+	- It seems that the devices and drives under "This PC" in Explorer are placed under
+	"/mnt" in the WSL terminal. So start by running `ls /mnt`. That will print out a
+	list of folders that will include "c" which is your normal "C://" drive, "wsl" which
+	contains system files related to WSL (you don't need to look at anything in here).
+	If you have a google drive, it will also be listed in here, we have seen it called
+	either "g" or "h".
+	- Now we can start digging into these drives to find the SSS files. If the files are
+	on google drive, start by running `ls /mnt/g` or `ls /mnt/h` depending on the name
+	of the drive you saw in the last step (if instead they are in the C drive, just
+	replace the "g" or "h" with "c"). That will list all the folders in your google
+	drive.
+	- We need to drill down into whichever listed folder contains the SSS data, so
+	we will keep appending the next folder name to the end of the path that we pass to
+	`ls` to step folder by folder until we find the files you want. **NOTE:** if you
+	spaces in names of your folders (which is common in Windows), we need to tell the
+	terminal that those spaces are part of the folder name, not marking a new parameter
+	to pass to `ls`. We can do that either by putting the entire path in quotation marks
+	or by "delimiting" the space by placing a backslash (`\`) directly before the space.
+	The terminal will actually add this delimiter for you if you use tab complete, so
```

Comment:
Readers likely won't know what "use tab complete" means yet (they're about to though, yay!)

Reply:
Consider separating this big paragraph into three bullets:
- parent bullet about drilling down
    - child about watching out for spaces in directory names
    - child about tab completion